### PR TITLE
docs: Desktop POC SDD

### LINKED ISSUE #1678: Prepare SDD for Desktop Prototype
1. Prepare a list of flank features for GUI.
2. Choose the most valuable one and also simple to implement.
3. Prepare mock screens and GUI for the chosen feature.
4. Prepare SDD for Desktop Prototype

This is a proof of concept so the solution should be simple to create and implement.

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Fixes #1678

SDD (without template) for PoC for Desktop

The design is just a mock (I am not good at it), so please do not complain about used colors 😃 

SDD

### CODE REVIEW

File: docs/desktop/flank_desktop.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Flank desktop prototype
+
+## Prototype Flank options
+
+The prototype will run simple Android test on given apk and test apk with option to specify some flanks flags, as well
```
Comment (Sloox) [inline docs/desktop/flank_desktop.md:5]:
```suggestion
The prototype will run a simple Android test on a given apk and test the apk with options to specify some flanks flags, as well
```

File: docs/desktop/flank_desktop.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+as max tests shards. Those flags are the most popular one based on [mixpanel analytics](https://mixpanel.com/report/2288623/view/2835119/dashboards#id=879189)
```
Comment (Sloox) [inline docs/desktop/flank_desktop.md:6]: analytics will change over time. Will we continually update this to always reflect the most common mixpanel analytics?

**Review by Sloox — CHANGES_REQUESTED** [review]

File: docs/desktop/flank_desktop.md (hunk 2 above)
Comment (piotradamczyk5) [reply docs/desktop/flank_desktop.md:6]: yes, but options chosen to create this POC based on this analytics

**Review by jan-gogo — APPROVED** [review]
Nice 👍, good job.

**Review by pawelpasterz — APPROVED** [review]

**Review by Sloox — APPROVED** [review]

**Pull request merged**